src/Components/Input.re

```diff
@@ -107,6 +107,8 @@ let%component make =
               (
                 ~theme,
                 ~style=Styles.default(~theme),
+                ~fontSize=18.,
+                ~fontFamily=Revery.Font.Family.fromFile("Roboto.Regular.ttf"),
```

Comment:
Is `Roboto.Regular.ttf` a typo? I see `Roboto-Regular.ttf` elsewhere.

Reply:
Indeed it is. Good catch!